### ISSUE TITLE
Adding text directly below a level 1 header in Markdown screws up the resulting Reveal.JS slideshow

### ISSUE THREAD

**Issue by AGresvig** [jgm/pandoc#816]
I noticed that if you have regular text directly following a level one (single #) header in the markdown input of a revealjs-slideshow, it completely screws up the section nesting.

Using the "habits.txt" example from the README it is simple to reproduce, simply insert text below the first header, like so:

```
% Habits
% John Doe
% March 22, 2005

# In the morning
Error

## Getting up

- Turn off alarm
- Get out of bed
...
```

Then compile a slideshow using the standard `pandoc -t revealjs -s habits.txt -o habits.html` and look at the resulting HTML, where the nesting of sections is screwed up.

Perhaps it is John's intention that top-level headings only be used to precede sub-headings (sub-slides), but it should probably either be fixed or documented in the README/man page, as I spent a small hour scratching my head over what was wrong with my markup.

Hope I haven't misunderstood anything here.

Thanks for a fantastic utility anyways!

**Comment by kenficara** [comment]
I don't know why this is so hard to explain, but the way Pandoc works obviates the entire point of a two-dimensional presentation. I'm working on one now that needs to have a 15-minute version and a 45-minute version. I would like the 15-minute version to consist of my level one slides, so that I could navigate through it horizontally, hitting the top-level points only. If questions come up about a particular point, I could navigate down to slides which have detailed examples of the points made on the first-level slide. For the full presentation I would navigate down through each subsection. This is pretty much the _point_ of a two-dimensional slide show. 
 
The way Pandoc works, the top level slides are just headings, and the two-dimensional slideshow degrades to slides where the heading shows up first, followed by the content.

**Comment by jgm** [comment]
That was a good explanation indeed!

**Comment by lafrech** [comment]
This was fixed in pandoc 2.7.

Related discussion : https://groups.google.com/forum/#!topic/pandoc-discuss/joqekZQRpq8
